test/metabase/fingerprinting_test.clj

```diff
@@ -0,0 +1,145 @@
+(ns metabase.fingerprinting-test
+  (:require (clj-time [coerce :as t.coerce]
+                      [core :as t])
+            [expectations :refer :all]
+            [metabase.fingerprinting :refer :all :as f]
+            [redux.core :as redux]))
+
+(def numbers [0.1 0.4 0.2 nil 0.5 0.3 0.51 0.55 0.22])
```

Comment:
I would still make these sorts of things `^:private` if they're not used in any other namespaces because it will make the code easier to work with in the future. For example if we want to rename, modify, or delete `numbers` we wouldn't have to worry about grepping the codebase to see if it was used anywhere else if it was marked `^:private`